runtimes/python/python.dockerfile

```diff
@@ -1,3 +1,9 @@
 ENV OPEN_RUNTIMES_ENTRYPOINT=main.py
+
 COPY requirements.txt .
-ENV FLASK_APP="/usr/local/server/src/server.py"
+
+# Create a virtual environment for the server dependencies
+RUN python3 -m venv server-env
+RUN server-env/bin/pip install --no-cache-dir -r requirements.txt
+
+ENV FLASK_APP="/usr/local/server/src/server.py"
```

Comment:
Do we still need this?

Reply:
We need the entry point - it is still server.py 
But the env variable name needs to be changed

Reply:
removed 👍